.scripts/modules.js

```diff
@@ -46,7 +46,7 @@ const execModules = async (command) => Promise.all(
 /* eslint-enable no-console */
 
 module.exports = {
-  npmArgument: JSON.parse(process.env.npm_config_argv || { original: '' }).original.join(' '),
+  npmArgument: JSON.parse(process.env.npm_config_argv || '{"original": []}').original.join(' '),
```

Comment:
@sasjo let me know if I should remove it from this PR. It's a bug but not related to my original goal here...